src/code/bank0.asm

```diff
@@ -4190,9 +4190,11 @@ label_27DD::
     pop  bc                                       ; $27E6: $C1
     jp   ReloadSavedBank                          ; $27E7: $C3 $1D $08
 
-label_27EA::
-    ld   a, $38                                   ; $27EA: $3E $38
+resetMusicFadeTimer::
```

Comment:
Nitpick: if this is a function, it should have a Capitalized name.

```suggestion
ResetMusicFadeTimer::
```

Reply:
need to be changed at all places. Could you do it please? I am working on a different branche at this moment

Reply:
Sure, no problem

Reply:
@zelosos I made the changes. But to update the PR, I need you to allow me to update your branch.

Can you check the ["Allow edits from maintainers"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) checkbox on this PR?

Reply:
was and is enabled as allways

Reply:
I commited your suggestion. maybe this is the root cause

Reply:
Oh indeed, that was my fault (a mismatch in the branch name).